src/Registers.cc

```diff
@@ -169,14 +169,11 @@ RegisterInfo<rr::X64Arch>::Table RegisterInfo<rr::X64Arch>::registers = {
   // The comparison for this is handled specially
   // elsewhere.
   RV_X64_WITH_MASK(ORIG_RAX, orig_rax, 0),
+  RV_X64_WITH_MASK(FS_BASE, fs_base, 8),
+  RV_X64_WITH_MASK(GS_BASE, gs_base, 8),
```

Comment:
To clarify: a mask of 8 doesn't make much sense. Using `RV_X64` means we'll die if `fs_base`/`gs_base` get out of sync during replay, but that's probably a good thing!